.github/workflows/run-tests.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Run poetry image
       uses: abatilo/actions-poetry@v2.0.0
       with:
-          poetry-version: 1.1.13
+          poetry-version: 1.6.1
```

Comment:
can we reintroduce the matrix option here like you did in the big PR about sphinx? much cleaner

Reply:
Is included in the latest dev-branch hotfix, as I had to fix the workflow anyway